modal/config.py

```diff
@@ -56,6 +56,10 @@
   Defaults to True.
   By default, Modal automatically mounts modules imported in the current scope, that
   are deemed to be "local". This can be turned off by setting this to False.
+* ``force_build`` (in the .toml file) / ``MODAL_FORCE_BUILD`` (as an env var).
+  Defaults to False.
+  By default, Modal automatically caches and re-uses container images. This can be
+  turned off by setting this to True. Meant to be used for debugging and profiling.
```

Comment:
I think we should probably not document this one, it's not meant for users to use.